portfolio.html

```diff
@@ -55,21 +55,21 @@ <h2 class="contentHeadings"><i class="fas fa-rocket" style="color:#B4535A"></i>
 		
 		
 		
-		<div class=contactBar>
+		<div class="contactBar">
 			<header class="header">
-				<img src=images/harryicon.jpeg class=profilePic alt="Harry smiling handsomely">
-				<h1>Harry Cusworth</h1>
+				<img src="images/harryicon.jpeg" class="profilePic" alt="Harry smiling handsomely">
+				<div class="name">Harry Cusworth</div>
 				<p class="status">Aspirational Developer
 				</p>
 			</header>
 					
-			<section class=contactDeets>
+			<section class="contactDeets">
 					<h3>Contact Details</h3>
-					<div class="contactUnit"><i class="fas fa-globe" style="color:#B4535A"></i><a href=https://harrycusworth.com/> Consultancy Home Page</a></div>
-					<div class="contactUnit"><i class="far fa-envelope" style="color:#B4535A"></i> <a href=mailto:harry.cusworth@protonmail.com> E-Mail</a></div>
+					<div class="contactUnit"><i class="fas fa-globe" style="color:#B4535A"></i><a href="https://harrycusworth.com/"> Consultancy Home Page</a></div>
+					<div class="contactUnit"><i class="far fa-envelope" style="color:#B4535A"></i> <a href="mailto:harry.cusworth@protonmail.com"> E-Mail</a></div>
 					<div class="contactUnit"><i class="fas fa-phone" style="color:#B4535A"></i>  0792*********</div>
-					<div class="contactUnit"><i class="fab fa-linkedin" style="color:#B4535A"></i><a href=https://www.linkedin.com/in/harry-cusworth-83b56a182/> Linkedin</a></div>
-					<div class="contactUnit"><i class="fab fa-github" style="color:#B4535A"></i> <a href=https://github.com/HarryCusworth>GitHub</a></div>
+					<div class="contactUnit"><i class="fab fa-linkedin" style="color:#B4535A"></i><a href="https://www.linkedin.com/in/harry-cusworth-83b56a182/"> Linkedin</a></div>
+					<div class="contactUnit"><i class="fab fa-github" style="color:#B4535A"></i> <a href="https://github.com/HarryCusworth">GitHub</a></div>
```

Comment:
the class 'contact unit' is used, but you have inline styling of the same text color in all the divs. put it in the css file so you only have to write/change it once should you wish too.